blueocean-commons/src/main/java/io/jenkins/blueocean/commons/BlueOceanConfigProperties.java

```diff
@@ -12,4 +14,7 @@
     public static final boolean ROLLBAR_ENABLED = Boolean.getBoolean(BLUEOCEAN_ROLLBAR_ENABLED_PROPERTY);
     public static final boolean BLUEOCEAN_FEATURE_JWT_AUTHENTICATION = Boolean.getBoolean(BLUEOCEAN_FEATURE_JWT_AUTHENTICATION_PROPERTY);
 
+    public static boolean isDevelopmentMode() {
```

Comment:
@imeredith where is the best place to define a system property for the ATH, so that I can check in the app if we are running in ATH?

Reply:
Honestly im not sure. Here?

Reply:
Or do you mean in the scripts? run.sh can pass it in during startup.